bpm/src/main/java/org/jboss/pnc/bpm/model/AnalyzeDeliverablesBpmRequest.java

```diff
@@ -31,6 +32,7 @@ public class AnalyzeDeliverablesBpmRequest implements Serializable {
     private static final long serialVersionUID = 2808475070463010184L;
 
     private final String operationId;
-    private final String milestoneId;
+    private final Optional<String> milestoneId;
```

Comment:
:grimacing: Optionals are usually for return values. And since I think this is a DTO, this would result in changing form `"milsestoneId": number` to `"milestoneId": {"present": boolean, "object": number}` (or something like that).